vocab-builder-core/src/main/java/com/github/tkurz/sesame/vocab/GenerationSetting.java

```diff
@@ -0,0 +1,53 @@
+package com.github.tkurz.sesame.vocab;
+
+import com.google.common.base.CaseFormat;
+
+public class GenerationSetting {
+
+	private String caseFormat;
+	private String constantPrefix;
+	private String constantSuffix;
+	public CaseFormat getCaseFormat() {
+		try {
+			if ( caseFormat != null ) {
+				return CaseFormat.valueOf(caseFormat);
+			}
+		} catch (Exception e) {
+		}
+		return null;
+	}
+	public void setCaseFormat(String caseFormat) {
+		this.caseFormat = caseFormat;
```

Comment:
It would be useful to do validation here rather than waiting until the get method is called to validate the input.